helm-chart/ray-cluster/README.md

```diff
@@ -78,6 +78,7 @@ helm uninstall raycluster
 | nameOverride | string | `"kuberay"` | String to partially override release name. |
 | fullnameOverride | string | `""` | String to fully override release name. |
 | imagePullSecrets | list | `[]` | Secrets with credentials to pull images from a private registry |
+| gcsFaultTolerance.enabled | bool | `false` |  |
```

Comment:
why this change?

Reply:
related changes: 
* https://github.com/ray-project/kuberay/pull/3881
* https://github.com/ray-project/kuberay/pull/3887

I think it's fine to keep this change here as long as we understand why it started to fail